### PR TITLE
When asking to publish show the location where it will be published

### PR DESCRIPTION
**Pull request by elijahmanor**

### CODE REVIEW

File: src/sequence-steps.js [hunk 1]
```diff
@@ -175,19 +176,21 @@ export function gitPushUpstreamMaster( [ git, options ] ) {
...
-		default: true
-	} ] ).then( answers => {
-		utils.log.begin( command );
-		if ( answers.publish ) {
-			return utils.exec( command ).then( data => utils.log.end() );
-		}
-		utils.log.end();
-	} );
+
+	console.log( "before utils.getPackageRegistry" );
```
Comment (Prestaul) [inline src/sequence-steps.js]: Was this an intentional log?

File: src/utils.js [hunk 2]
```diff
@@ -61,6 +62,19 @@ export default {
 			} );
 		} );
 	},
+	getPackageRegistry() {
+		const pkg = this.readJSONFile( "./package.json" );
+		const registry = get( pkg, "publishConfig.registry" );
+
+		if ( registry ) {
+			return new Promise( resolve => resolve( registry ) );
```
Comment (Prestaul) [inline src/utils.js]: You can use `Promise.resolve( registry )` as well.

File: src/sequence-steps.js (hunk 1 above)
Comment (elijahmanor) [inline src/sequence-steps.js]: doh, was an oversight

File: src/utils.js (hunk 2 above)
Comment (elijahmanor) [inline src/utils.js]: great tip, didn't realize that! i'll update

**Pull request merged**